docs/flow/modin/experimental/reshuffling_groupby.rst

```diff
@@ -0,0 +1,70 @@
+Reshuffling GroupBy
+"""""""""""""""""""
+
+The experimental GroupBy implementation utilizes Modin's reshuffling mechanism that gives an
+ability to build range partitioning over a Modin DataFrame.
+
+In order to enable/disable this new implementation you have to specify ``cfg.ExperimentalGroupbyImpl``
+:doc:`configuration variable: </flow/modin/config>`
+
+.. code-block:: ipython
+
+    In [4]: import modin.config as cfg; cfg.ExperimentalGroupbyImpl.put(True)
+
+    In [5]: # past this point Modin will always use the new reshuffling groupby implementation
+
+    In [6]: cfg.ExperimentalGroupbyImpl.put(False)
+
+    In [7]: # past this point Modin won't use reshuffling groupby implementation no more
```

Comment:
```suggestion
    In [7]: # past this point Modin won't use reshuffling groupby implementation more
```

Reply:
I'd say
In [7]: # past this point Modin won't use reshuffling groupby implementation anymore